### PR TITLE
Front end fixes

### LINKED ISSUE #1004: If there is only one product type don't ask about it
It is pretty annoying to always click the same value:
image

### PR DESCRIPTION
**Pull request by weronka**
Fix #1004

### CODE REVIEW

File: saleor/static/scss/layouts/_order-details.scss [hunk 1]
```diff
@@ -26,4 +29,16 @@
...
+  &__total {
+    padding-top: $global-padding;
+    color: $skull-gray;
+  }
+  &__last-row {
+    border-bottom-color: $skull-gray; 
+  }
+  &__table-footer {
+    @include media-breakpoint-up(sm) {
+      padding-left: 5.75rem;
```
Comment (benekex2) [inline saleor/static/scss/layouts/_order-details.scss]: use $global-padding

**Comment by weronka** [comment]
@benekex2 this country select will be replaced there #983

**Comment by weronka** [comment]
Print screens from new style guide template:

screenshot 2
screenshot 1

**Comment by weronka** [comment]
@patrys could you have a look, please

**Pull request merged**